.pre-commit-config.yaml

```diff
@@ -0,0 +1,26 @@
+repos:
+  -   repo: https://github.com/PyCQA/isort
+      rev: 5.6.4
+      hooks:
+      - id: isort
+  -   repo: https://gitlab.com/pycqa/flake8
+      rev: 3.8.4
+      hooks:
+      - id: flake8
+  -   repo: https://github.com/pre-commit/pre-commit-hooks
+      rev: v3.3.0
+      hooks:
+      -   id: trailing-whitespace
+      -   id: end-of-file-fixer
+      -   id: check-case-conflict
+      -   id: check-json
+  -   repo: https://github.com/pre-commit/mirrors-eslint
+      rev: '7.15.0'
+      hooks:
+      -   id: eslint
```

Comment:
Does using this pre-commit hook mean we should remove the eslint from `package.json`? I imagine we do not want them to go out of sync with one another (which they already are). Using pre-commit as the single source of truth may be simpler depending on how Jenkins will be configured in the end.

```
package.json
5        "pretest": "eslint django/ js_tests/admin/ js_tests/gis/",
12       "eslint": "^6.8.0",
```